presto-main/src/main/java/com/facebook/presto/operator/scalar/annotations/ScalarImplementation.java

```diff
@@ -165,16 +170,16 @@ public ScalarImplementation(
         return clazz;
     }
 
-    private static boolean isParameterNullable(Class<?> type, boolean nullableArgument, boolean nullFlag)
+    private static boolean isParameterNullable(Class<?> type, NullConvention nullConvention)
```

Comment:
give this function (and the variable the return value this function is assigned to) a better name